documentation/source/repurchase-agreement-representation.rst

```diff
@@ -0,0 +1,857 @@
+.. include:: links.rst
+
+Repurchase Agreement Representation
+==================================
+
+Background
+------------
+The ICMA CDM initiative is extended to support the digitalization of the 
+repo market through the adoption of a standardized domain model and associated 
+lifecycle events based on the ERCC Best Practices Guide and industry standards.  
+The ICMA promotes adoption of CDM by organizing working groups and workshops on 
+proof-of-concept projects and integration.
+
+The repo elements of the model were designed with participation and contribution 
+of the ICMA CDM for Repo and Bonds Steering Committee.
+
+
+Introduction
+------------
+The purpose of the CDM for repo and bonds is to provide standardized digital 
+representation of repo products and lifecycle events that will reduce reconciliation 
+failures caused by data and processing mismatches and provides firms with a software 
+toolkit that can be incorporated into existing systems or established as the baseline 
+for a new platform.
+
+Repurchase Agreements are represented in the CDM as contractualProducts
+with an InterestRatePayout and an AssetPayout. Purchase date and repurchase
```

Comment:
This writing of CDM object names (types, functions, attributes etc.) such as "InterestRatePayout" or "Asset Payout" contravenes the doc style guide - see: https://cdm.docs.rosetta-technology.io/source/contribution.html#special-format-for-cdm-objects. They should either be `InterestRatePayout` (code quote) or simply "interest rate payout". There are a many instances of this in the proposed doc.

Reply:
@tomhealey-icma This comment is still outstanding. The writing of object names in this file does not conform to the style guide and needs to be adjusted.